src/theory/quantifiers/sygus/sygus_grammar_cons.cpp

```diff
@@ -1238,7 +1238,9 @@ void CegGrammarConstructor::mkSygusDefaultGrammar(
         sdts[iat].d_sdt.addConstructor(op, ssop.str(), opCArgs, 0);
       }
     }
-    if (polynomialGrammar)
+    // in the rare case that we have no arithmetic terms and are using
+    // polynomial grammar, we use the any constant constructor below instead.
```

Comment:
```suggestion
    // polynomial grammar, we use the "any constant" constructor below instead.
```
I think this makes it clearer.